Education.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace MarsQA_1.Pages
+{
+    class Education

```

Comment:
A bit overkill to create a class just for this feature. 
You could add this into a class that is testing the Profile page and group them in regions

Reply:
You are right but I was just trying to follow the trend in the given framework so that it would be easy to understand and easy to reuse. For example: for Language class I have written only one test case (Adding language to the language tab) as of now. What if in future I have to write more test cases for language tab alone. For example trying to add a 5th language or adding a language that already exist or trying to delete an existing language. I think creating different classes would be easy to manage in that situation. What do you think about it?